.github/workflows/release-notifier.yml

```diff
@@ -0,0 +1,33 @@
+name: Create Issue On Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create Issue
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.DOCS_PAT }}  # Assuming 'PAT' is your Personal Access Token stored as a secret
```

Comment:
same questions as the other repo - whose is this and how is it managed? what shall we do in case it expires and breaks the build?

Reply:
it's mine - and if it breaks the build, we remove the notifier. Doesn't have an expiry.